### PR TITLE
Add roadmap to README.md

### PR DESCRIPTION
**Pull request by LucasSte**
I summarized our roadmap to prioritize items that make sense for developers. I also included low-hanging fruits for first-timers who are willing to contribute.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -119,6 +119,37 @@ You will have a file called flipper.contract. You can use this directly in
 the [Polkadot UI](https://substrate.dev/substrate-contracts-workshop/#/0/deploy-contract),
 as if your smart contract was written using ink!.
 
+## Roadmap
+
+Solang has a long way to go in order to achieve full compatibility for the blockchains we target.
```
Comment (seanyoung) [inline README.md:124]: I don't think we should downplay what solang can already do. It's really not *that* far from full compatibility.

File: README.md [hunk 2]
```diff
@@ -119,6 +119,37 @@ You will have a file called flipper.contract. You can use this directly in
 the [Polkadot UI](https://substrate.dev/substrate-contracts-workshop/#/0/deploy-contract),
 as if your smart contract was written using ink!.
 
+## Roadmap
```
Comment (seanyoung) [inline README.md:122]: This should be "Tentative Roadmap" (or something like that)

File: README.md [hunk 3]
```diff
@@ -119,6 +119,37 @@ You will have a file called flipper.contract. You can use this directly in
...
+### V0.3
+
+| Milestone | Status |
+| --------- | ------ |
+| Call Rust contracts from Solidity | Not started |
+| Parse and resolve inline assembly | In progress |
+
+### Contributions are welcome!
+
+These are the low-hanging fruits for first-time contributors:
```
Comment (seanyoung) [inline README.md:143]: Ideally we should have a "first-time issues" label and bunch of issues with those labels.

File: README.md [hunk 4]
```diff
@@ -119,6 +119,37 @@ You will have a file called flipper.contract. You can use this directly in
...
+| Specify values as "1 sol" and "1e9 lamports" | Not started |
+| Solana SPL tokens compatibility | In progess |
+
+### V0.3
+
+| Milestone | Status |
+| --------- | ------ |
+| Call Rust contracts from Solidity | Not started |
+| Parse and resolve inline assembly | In progress |
+
```
Comment (seanyoung) [inline README.md:140]: Any reason for not including the other versions?

**Review by seanyoung — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (LucasSte) [reply README.md:143]: Should I remove this section?

File: README.md (hunk 4 above)
Comment (LucasSte) [reply README.md:140]: The readme would get too long, and I don't think it should.

File: README.md (hunk 3 above)
Comment (seanyoung) [reply README.md:143]: I think we should.

Having said that we should definitely create a bunch of "good first issue" for folks to contribute.

File: README.md (hunk 4 above)
Comment (seanyoung) [reply README.md:140]: ok

**Review by seanyoung — APPROVED** [review]

**Pull request merged**